### PR TITLE
Fix GCC 4.8 support and C++>11 usage

### PR DESCRIPTION
**Pull request by strager**

### CODE REVIEW

**Comment by strager** [comment]
For some reason I can't add @mnovakovic as a reviewer. =\

**Comment by mnovakovic** [comment]
lgtm

File: Sources/main.cpp [hunk 1]
```diff
@@ -260,7 +260,7 @@ void PrintUsage(const char *programName)
             "  -f -            specify a mapping file through standard input\n"
             "  -h              show this usage text and exit\n"
             "  -b              produce APPXBUNDLE instead of APPX\n"
-            "  -o output-file  write the APPX (or APPXBUNDLE if -b is specified) "
+            "  -o output-file  write the APPX (or APPXBUNDLE if -b is specified)\n"
```
Comment (mnovakovic) [inline Sources/main.cpp:263]: This is already done in the other pull request?

Comment (strager) [reply Sources/main.cpp:263]: Yeah. GitHub did not coalesce my PRs.

**Pull request merged**